chunk/chunk_store.go

```diff
@@ -332,25 +338,45 @@ func (c *AWSStore) CreateTables() error {
 	return err
 }
 
-func bigBuckets(from, through model.Time) []int64 {
+func bigBuckets(from, through model.Time, dailyBucketsFrom model.Time) []string {
```

Comment:
I'd make this a method on `AWSStore` and not pass `dailyBucketsFrom` as a param.